base_layer/core/src/base_node/state_machine_service/state_machine.rs

```diff
@@ -52,8 +52,6 @@ const LOG_TARGET: &str = "c::bn::base_node";
 pub struct BaseNodeStateMachineConfig {
     pub blockchain_sync_config: BlockchainSyncConfig,
     /// The maximum amount of VMs that RandomX will be use
-    // TODO: This is a potential conflict with 'BaseNodeConfig::max_randomx_vms'
-    pub max_randomx_vms: usize,
```

Comment:
Github wont let me comment lower but can we remove `bypass_range_proof_verification` below?